cpp/RuntimeDecorator.cpp

```diff
@@ -23,12 +23,7 @@ void injectJSIBindings(jsi::Runtime &rt) {
       jsi::PropNameID::forAscii(rt, "jsi_registerMarkdownWorklet"),
       1,
       [](jsi::Runtime &rt, const jsi::Value &thisValue, const jsi::Value *args, size_t count) -> jsi::Value {
-#ifdef WORKLETS_INSTALLED
-        auto worklet = extractSerializableOrThrow<SerializableWorklet>(rt, args[0]);
-#else
-        auto worklet = extractShareableOrThrow<SerializableWorklet>(rt, args[0]);
-#endif
-        auto parserId = registerMarkdownWorklet(worklet);
+        auto parserId = registerMarkdownWorklet(extractSerializableOrThrow<SerializableWorklet>(rt, args[0]));
```

Comment:
Let's add `const` in this PR or in a separate PR.
```suggestion
        const auto parserId = registerMarkdownWorklet(extractSerializableOrThrow<SerializableWorklet>(rt, args[0]));
```